CI/batch/docker/Dockerfile.gpu

```diff
@@ -10,6 +10,8 @@ RUN apt-get update \
  && apt-get autoremove -y \
  && apt-get clean
 
+RUN apt install tesseract-ocr -y
```

Comment:
Do we need to update the tutorial since it is not shipped with our binary?

Reply:
@cheungdaven